libs/content/workspace-ffi/src/android/api.rs

```diff
@@ -215,6 +215,7 @@ pub extern "system" fn Java_app_lockbook_workspace_Workspace_openDoc(
     obj.workspace.open_file(id, new_file == 1, true);
 }
 
+// todo: can't close non-file tabs (mind map)
```

Comment:
todo: review ffi functions that may assume every tab has an id with @smailbarkouch